Docs/source/usage/parameters.rst

```diff
@@ -2037,6 +2037,12 @@ Particle push, charge and current deposition, field gathering
 
     Note that this input parameter is not optional and must always be set in all input files provided that there is at least one particle species (set in input as ``particles.species_names``) or one laser species (set in input as ``lasers.names``) in the simulation. No default value is provided automatically.
 
+* ``warpx.nguards`` (`integers, one per space dimension`, optional)
```

Comment:
Should this be called `warpx.deposition_nguards`, because it only overwrites the deposition arrays?

Or should it also influence the other multi-fabs?